src/formatter/format.zig

```diff
@@ -66,6 +67,20 @@ fn checkFile(dir: std.fs.Dir, filePath: []const u8) !void {
 				lineStart = false;
 			},
 			'\t' => {},
+			'.' => {
+				var endIndex = i + 1;
+				if(isZon) {
+					while(endIndex < data.len) : (endIndex += 1) {
+						if(data[endIndex] == '_') {
+							printError("Zon entry names must be in camelCase format!", filePath, data, i + 1);
```

Comment:
This could cause problems if we have want to have an id on the left hand side of an argument.

I think it would be preferable to enforce the naming conventions in the Zig code base instead, since most of these directly relate to variable names.